app/src/main/java/space/taran/arknavigator/mvp/presenter/adapter/tagsselector/TagsSelectorPresenter.kt

```diff
@@ -20,11 +20,15 @@ class TagsSelectorPresenter(
     private var index: ResourcesIndex? = null
     private var storage: TagsStorage? = null
     private val actions = ArrayDeque<TagsSelectorAction>()
+    private var filter = ""
+    private var filterEnabled = false
 
     var included = mutableSetOf<Tag>()
         private set
     var excluded = mutableSetOf<Tag>()
         private set
+    private var available = listOf<Tag>()
```

Comment:
Do we really need both fields `available` and `availableTags`?
They look almost the same and easy to confuse.